docs/src/pages/[...docPath].tsx

```diff
@@ -64,6 +77,16 @@ interface IDocPage {
 }
 
 const DocPage: NextPage<IDocPage> = ({ source, frontMatter }): JSX.Element => {
+  const router = useRouter();
+
+  useEffect(() => {
+    const documentationView = document?.getElementById(
+      "documentation-view"
```

Comment:
why we can't use React useRef API instead? For that we would need to move this logic to DocumentLayout but for me it looks reasonable (e.g. if we will add another page with the same layout we wouldn't need to duplicate this logic)

Reply:
@RavenInd

Reply:
corrected